### ISSUE TITLE
invalid entries detected

### ISSUE THREAD

**Issue by artlessknave** [batstyx/evemon-alpha-skill-plans#12]
"reactions planned to 1"
evemon 3.0.3

**Comment by batstyx** [comment]
This is **Working as Intended™**.

The long version of these notes are for my own reference as much as anything.

### TLDR Version
The `Reactions` skill isn't currently included in EVEMon. Updating the skill tree requires a new EVEMon data release. [Take it up with them](https://github.com/evemondevteam/evemon/issues/new) ;)

### Long Version
The plans are technically correct but EVEMon doesn't currently include `Reactions` in it's skills file<sup>[1](#myfootnote1)</sup>. The *Invalid Entries Detected* message will be displayed on opening the plan for the first time after import while that's the case. 

With some technical know-how, it's possible to add `Reactions` [xml](https://gist.github.com/batstyx/fd7fce1b6f678ee69421793963e71556) to the skills file<sup>[1](#myfootnote1)</sup> manually, but this causes the EVEMon startup check to prompt to download the latest version of the file as defined in the [patch notes](https://bytebucket.org/EVEMonDevTeam/evemon/wiki/updates/patch.xml).

EVEMon needs a new data release for this to be resolved. Having looked at the code-base it appears these are generated from a SQL data dump from CCP (is that even still a thing‽<sup>[3](#myfootnote3)</sup>) rather than from the API, which may explain why it hasn't been kept up-to-date. 
Alternatively they may be distracted by the imminent [XML API shutdown](https://developers.eveonline.com/blog/article/important-information-on-the-future-of-the-eve-api).

<a name="myfootnote1"><sup>1</sup></a> `eve-skills-{language code}.xml.gzip`<sup>[2](#myfootnote2)</sup> installed in `%PROGRAMFILES(x86)%\EVEMon\Resources\` then used in `%APPDATA%\EVEMon\`
<a name="myfootnote2"><sup>2</sup></a> language code is en-US by default
<a name="myfootnote3"><sup>3</sup></a> Narrator: [It is not](https://forums-archive.eveonline.com/message/6794506/)
